src/scancode/cli.py

```diff
@@ -223,20 +232,39 @@ def reindex_licenses(ctx, param, value):
 number of files processed. Use --verbose to display file-by-file progress.
 '''
 
+formats = ('json', 'json-pp', 'html', 'html-app', 'spdx-tv', 'spdx-rdf')
```

Comment:
But to actually do anything meaningful you should start to add a test ASAP... otherwise you are adding things that do not have much usage for now.